### ISSUE TITLE
Top Menu doesn't have sufficient contrast on Focus or Hover states

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#5]
**URL**: https://hcdev.openplus.ca/en
**User Agent:** Chrome/Firefox
**Language**: both

**Issue Description:** The menu on top doesn't have enough contrast with focus or hover with it's adjacent colours. For example, when the user has focus on Home, the menu item next to it is People. The contrast between the two is 1.41:1. It should be minimum 3:1. 
![image](https://user-images.githubusercontent.com/65728700/126988972-e14a7241-0b15-4e41-b483-5efcca607ad0.png)

**Steps to reproduce:** Visual, keyboard tabbing, mouse hovering
**Expected results:** The contrast of the focused items background should be minimum 3:1 with the adjacent colour.

**Comment by echouin** [comment]
From OP: This one is done and can be seen on ﻿the live website. Increased the white colour for the hover so that the contrast is in line with the proper ratios
![MenuContrast (3)](https://user-images.githubusercontent.com/88099454/157729168-2d16df52-2641-4064-9239-0ef8d574d478.png)
.

**Comment by ndolter** [comment]
The color contrast (black beside the hovered state of dark grey) is still failing with a contrast ratio of 2.2:1.
The requirement calls for a 3:1 contrast ratio against adjacent colors for 2 types of non-text content:
1. Active user interface components
     o Any visual boundary that indicates the active component's clickable area must have a contrast ratio of at least 3:1.
     o Any visual effect that indicates the active component's state must have a contrast ratio of at least 3:1.

**Comment by hydegeoff** [comment]
Take a look at the screenshot.  The arrows in red are problematic, and we recommend using the colour that is being pointed to in green below.    This lighter grey rgb(109, 110, 113) meets all requirements, Note: the grey cannot be too light, because white letters of the username and light grey need to be 4.5:1 contrast ratio.
![Menu Colour Background does not meet the WCAG](https://user-images.githubusercontent.com/88147013/160653050-a8181a5c-7626-4fe2-a7e9-15100f9a427d.jpg)

**Comment by echouin** [comment]
@hydegeoff from OP:  have reported a new Drupal core issue here:
﻿﻿https://www.drupal.org/project/drupal/issues/3273789
﻿
﻿There is already a couple very active and very closely related threads here that may be followed as well.
﻿
﻿﻿https://www.drupal.org/project/drupal/issues/3097907
﻿﻿https://www.drupal.org/project/drupal/issues/3270230

**Comment by hydegeoff** [comment]
@echouin It looks like Drupal has made a fix. Can you please confirm the fix has being applied to RSAMS in Dev environment?

**Comment by echouin** [comment]
@hydegeoff OP has confirmed that they do keep Drupal up-to-date. Please verify on Prod.
